recipes/vtki/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "vtki" %}
+{% set version = "0.17.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 295b49f802b4a511d3ff44cf88aee6626507aa3be58222c0725cd989e2d46b1a
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - numpy
```

Comment:
The `build` requirements should probably contain only `{{ compiler('c') }}`.

This might also fix the Windows error.